tests/dragonfly/cluster_test.py

```diff
@@ -1063,38 +1071,14 @@ async def generate_config():
         keeping = node.slots[num_outgoing:]
         node.next_slots.extend(keeping)
 
-    # some more time fo seeder
-    await asyncio.sleep(1.0)
-
     seeder.stop()
     await fill_task
+    # wait to be sure that migrations is done
     await asyncio.sleep(1.0)
```

Comment:
The comment used to be `# some more time fo seeder`... So now we wait for it wo be done? Why 1s?

Let's better do it like control plane would do it, checking the status and blocking right until it's finished

Reply:
Actually, where did the checking loop disappear, seems to be an earlier commit 😆